website/docs/configuration/debug-prompt.mdx

```diff
@@ -49,6 +49,7 @@ properties below - defaults to `[DBG]: `
 - `.Shell`: `string` - the current shell name
 - `.UserName`: `string` - the current user name
 - `.HostName`: `string` - the host name
+- `.Code`: `int` - the last exit code
```

Comment:
this will only be valid for PowerShell now, right?

Reply:
Yes, since the debug prompt feature is only available for PowerShell as is described at the beginning of the page.

> This feature only works in `powershell` for the time being.